apps/next/src/app/components/nav/nav.ts

```diff
@@ -27,31 +32,17 @@ import { ThemeService } from '../../theme.service';
   },
 })
 export class Nav {
+  _navData$ = this._pageService._getCategories().subscribe((data) => {
+    this._navItems.push(...data);
```

Comment:
Why do you need to push these, are these not all categories that are coming in? 
Maybe we do not need to subscribe in the component at all?

Reply:
Regarding the subscription, I've tried to get the categories with a http get request and to return them as an array to no avail. Anything other the `{ responseType: 'json' }` returns an ArrayBuffer and I would rather not use that in our service.

Reply:
The `responseType: 'json'` is fine, but you could this here:
```suggestion
    _navData$ = this._pageService._getCategories();
```

and use this in the template like this:
```html
<li class="next-nav-item" *ngFor="let item of (_navData | async)">
```